docs/example_static_html.md

```diff
@@ -195,6 +195,14 @@ You can try the server here:
 Obviously, our busybox example is very silly. We just wanted to show you how
 Tilt can work with any language, even a silly one.
 
+Once you're done, set up a CI test to ensure it doesn't break!  The example
+repo uses [`ctlptl`](https://github.com/tilt-dev/ctlptl) to create a single-use
```

Comment:
FWIW, this confused me because it wasn't apparent that all this was happening in CI. Consider something like:
"In the example repo, on each PR, CircleCI will use ctlptl to create... then run tilt CI to deploy the services and ensure that they stand up correctly"

Reply:
done